src/main/java/org/jitsi_modified/impl/neomedia/rtp/remotebitrateestimator/RemoteBitrateEstimatorAbsSendTime.java

```diff
@@ -188,34 +180,45 @@ public RemoteBitrateEstimatorAbsSendTime(
         this.lastUpdateMs = -1;
     }
 
+    /**
+     * Reset back to the state immediately after construction.
+     */
+    public void reset()
+    {
+        remoteRate.reset();
+        this.incomingBitrate = new RateStatistics(kBitrateWindowMs, kBitrateScale);
+        this.incomingBitrateInitialized = false;
+        this.firstPacketTimeMs = -1;
+        this.lastUpdateMs = -1;
+    }
+
     /**
      * Notifies this instance of an incoming packet.
      *
+     * @param nowMs the current time when this method is called
      * @param arrivalTimeMs the arrival time of the packet in millis.
-     * @param sendTime24bits the send time of the packet in AST format
+     * @param sendTimeMs_ the send time of the packet in millis
      * (24 bits, 6.18 fixed point).
      * @param payloadSize the payload size of the packet.
      * @param ssrc the SSRC of the packet.
      */
     @Override
     public void incomingPacketInfo(
+        long nowMs,
         long arrivalTimeMs,
-        long sendTime24bits,
+        long sendTimeMs_,
```

Comment:
why the trailing underscore here?

Reply:
I pass in sendTimeMs_, which gets round-tripped via sendTime24Bits to sendTimeMs.

I _think_ (though I'm not 100% sure) that this ends up doing a calculation modulo 2^32, which is important.

Reply:
sorry, i just meant that i don't think we put trailing underscores on function argument names